components/led_strip/led_strip.c

```diff
@@ -20,6 +21,10 @@
 #include "freertos/task.h"
 
 #include <string.h>
+#include "driver/gpio.h"
+#include "esp_log.h"
+
+#define TAG "led_strip"
```

Comment:
it's better to make that a static const char. I think the compiler/linker merges identical strings, but I'm not sure.

Reply:
Done